planner/core/testdata/integration_serial_suite_out.json

```diff
@@ -2821,23 +2821,23 @@
       {
         "SQL": "desc format='brief' select /*+ use_index_merge(t) */ * from t where a =1 or (b=1 and length(b)=1)",
         "Plan": [
-          "IndexMerge 1.72 root  ",
+          "IndexMerge 8.00 root  ",
           "├─IndexRangeScan(Build) 1.00 cop[tikv] table:t, index:a(a) range:[1,1], keep order:false",
-          "├─Selection(Build) 0.80 cop[tikv]  eq(length(cast(1, var_string(20))), 1)",
+          "├─Selection(Build) 0.80 cop[tikv]  1",
```

Comment:
This change is expected, because after fixing the bug, the constant propagate can infer that
`b = 1 and length(b) = 1` is always true.
So the this filter condition changes and the cost of the plan changes...